charts/common/tests/pdb_test.yaml

```diff
@@ -60,7 +43,6 @@ tests:
           value: "27%"
```

Comment:
There are several tests here that haven't got any assertions. What's their value?

Reply:
from - values